### DISCUSSION TITLE
Set default arguments when committing

### DISCUSSION THREAD

**Discussion by lunarythia** [magit/magit#5010] [Support]
Apologies if this question has already been asked as I couldn’t find anything.

I’m trying to enable the `-S` argument for committing by _default_.

![圖片](https://github.com/magit/magit/assets/63614345/c771655b-d6b1-4395-a526-94eb09843d4c)

I tried doing both `C-x s` and `C-x C-s`, neither of which saved Sign using gpg to be enabled by default.

**Comment by tarsius** [comment]
There was a regression in transient but I fixed it yesterday. Update transient and it should work again.
